emmet-builders/emmet/builders/materials/ml.py

```diff
@@ -55,8 +55,8 @@ def __init__(
             model_name = {"chgnetcalculator": "chgnet"}.get(model_name, model_name)
             pkg_name = {"m3gnet": "matgl"}.get(model_name, model_name)
             self.provenance = dict(
-                model_name=model_name,
-                model_version=version(pkg_name),
+                name=model_name,
```

Comment:
is just `model` ok? clearer what it refers to than `name` imo

Reply:
Makes sense to me! Changing to `model` also doesn't result in any runtime warnings, so should be all good.